app/services/embedder.rb

```diff
@@ -1,27 +1,9 @@
 class Embedder
-  def initialize(embedding_model, traceable: nil)
-    @embedding_model = embedding_model
-    @traceable = traceable
-  end
+  include Helpers::ModelClient
```

Comment:
I'd avoid mixins here as well, if possible.

In general, they make testing harder and make your class behave in ways that are not obvious from the code in the class itself. It's better to have collaborators than mixins.

Reply:
Makes sense. Done.